### PR TITLE
Add info.smplayer.SMPlayer

### PR DESCRIPTION
**Pull request by smplayer-dev**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
-  `--filesystem=host` so that the user can play any file (and save subtitles) anywhere in the file system.
- `--talk-name=org.freedesktop.ScreenSaver` to turn off power saving during playback.
- ` --own-name=org.mpris.MediaPlayer2.smplayer` for mpris support

- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build info.smplayer.SMPlayer

**Comment by smplayer-dev** [comment]
bot, build info.smplayer.SMPlayer

**Comment by nedrichards** [comment]
Is there anyone else from upstream who will need access as well?

File: info.smplayer.SMPlayer.yml [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+rename-desktop-file: smplayer.desktop
+command: smplayer
+finish-args:
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=x11'
+  - '--socket=fallback-x11'
```
Comment (nedrichards) [inline info.smplayer.SMPlayer.yml:13]: you only need this permission if you're enabling wayland as well (and it's used to select wayland by default). If you're not enabling wayland I'd remove it.

File: flathub.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: this is disappointing.

File: info.smplayer.SMPlayer.yml (hunk 1 above)
Comment (hfiguiere) [reply info.smplayer.SMPlayer.yml:13]:
```suggestion
```

**Comment by barthalion** [comment]
bot, build info.smplayer.SMPlayer

**Comment by barthalion** [comment]
bot, build info.smplayer.SMPlayer

**Comment by barthalion** [comment]
bot, build info.smplayer.SMPlayer

**Comment by smplayer-dev** [comment]
> Is there anyone else from upstream who will need access as well?

No, just me.

**Comment by smplayer-dev** [comment]
> Build [57007 successful](https://flathub.org/builds/#/builders/32/builds/57007)

It's funny, it failed for me building it without mplayer at github:
https://github.com/smplayer-dev/flatpak-builds/runs/3392892463

Maybe it's possible to compile mplayer by not using `--enable-runtime-cpudetection` on aarch64.

**Comment by barthalion** [comment]
Weird, that build failure above doesn't seem related to mplayer at all at first glance.

Let's see if it still builds:
bot, build info.smplayer.SMPlayer

**Comment by barthalion** [comment]
bot, build info.smplayer.SMPlayer

**Comment by barthalion** [comment]
bot, build info.smplayer.SMPlayer

**Comment by barthalion** [comment]
Doh, I can see now why you used the simple buildsystem instead. I will switch back to it.

**Comment by barthalion** [comment]
bot, build info.smplayer.SMPlayer

**Comment by barthalion** [comment]
bot, build info.smplayer.SMPlayer

**Pull request closed without merging**